packages/hooks/src/useCollectionQuery/useCollectionQuery.test.tsx

```diff
@@ -0,0 +1,345 @@
+import { act, renderHook } from "@testing-library/react-hooks";
+import { useCollectionQuery } from "./useCollectionQuery";
+import {
+  LIST_QUERY,
+  ListQueryType,
+  SUBSCRIPTION_QUERY,
+  SubscriptionQueryType,
+  buildListRequestMock,
+  buildListRequestMockForNextPage,
+  buildSubscriptionRequestMock,
+  listQueryResponseMock,
+  setListQueryMockHasNextPage,
+  subscriptionQueryMock,
+  wait,
+  wrapper,
+} from "./__mocks__";
```

Comment:
This doesn't quite fit mocks. Which are normally used to mock modules.
https://jestjs.io/docs/manual-mocks

It's totally cool to have utility methods like you have here but, in that case I don't think they fit with `__mocks__`

Reply:
Would you prefer a `__test__` directory next to the `mock` directory? Happy to do that, just figured it wans't worth the extra overhead.

Reply:
I'll go with calling it `__test_utils__` for now unless I hear back about a better name.